### DISCUSSION TITLE
[Suggestion] Ability to remap F keys with 'Forward' and 'Back' mouse buttons on "Key Remapping"

### DISCUSSION THREAD

**Discussion by ZeiZeiii** [runelite/runelite#16276] [Ideas]
Title

**Comment by zahti** [comment]
Looks like this was answered as "**wontfix**" here: https://github.com/runelite/runelite/discussions/16122

Note: I would also like to have this as an option as well.

**Comment by leestran1995** [comment]
Thirding this suggestion, seems right in line with what Jagex allows with regards to key remapping, don't understand why it's such an issue
